apps/prairielearn/src/ee/pages/instructorAiGenerateDrafts/SampleQuestionDemo.tsx

```diff
@@ -154,7 +154,7 @@ export function SampleQuestionDemo({
 
   useEffect(() => {
     handleGenerateNewVariant();
-  }, [promptId]);
+  }, [promptId, handleGenerateNewVariant]);
```

Comment:
Did you see the warning that this caused? IMO we should fix that and also see about elevating those warnings to errors.

Reply:
I didn't see the warning in VSCode, but I did in my terminal. I elevated this to an error in [7755746](https://github.com/PrairieLearn/PrairieLearn/pull/12369/commits/77557462a1c3fec5b7e11f06456cae199ffa0c7f).